### PR TITLE
[image integrity] add image integrity disable

### PR DESCRIPTION
**Pull request by fseldow**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
`az aks update --disable-image-integrity`

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
image integrity

File: src/aks-preview/setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.5.149"
+VERSION = "0.5.150"
```
Comment (FumingZhang) [inline src/aks-preview/setup.py:12]: 150 has been used, if you want to release a new version, please update to 151.

Please also update HISTORY.rst and leave some notes.

**Review by FumingZhang — COMMENTED** [review]
The change looks good to me, could you please add a live test case for this new option? May refer to the test case of image cleaner

**Comment by fseldow** [comment]
> The change looks good to me, could you please add a live test case for this new option? May refer to the test case of image cleaner

added

**Review by FumingZhang — APPROVED** [review]
LGTM

[live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=77802510&view=results) validation passed

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**